PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please add a line in the relevant section of [CHANGELOG.md](https://github.com/ImperialCollegeLondon/virtual_rainforest/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Optimization (back-end change that speeds up the code)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+
+
+# Key checklist:
+
+- [ ] No style issues: `$ flake8`
+- [ ] All tests pass: `$ python run-tests.py --unit`
```

Comment:
There is no file `run-tests.py`. To run the tests, users will need to run `pytest` instead.